File: Java-Python-C++/coj.uci.cu/2415 - Find the Wedding Cake Volume/Main.java
/*   Problem:  Find the Wedding Cake Volume
 *   Code: 2415
 *   Site: coj.uci.cu
 *   Status: WA
 *   type: Geometria
 */


import java.util.*;
import java.math.*;
import java.text.DecimalFormat;
import java.text.DecimalFormatSymbols;

public class Main
{
	
	public static void main (String args[])
	{
		DecimalFormatSymbols simbolos = new DecimalFormatSymbols();
        simbolos.setDecimalSeparator('.');
        DecimalFormat formateador = new DecimalFormat("####.##",simbolos);
		Scanner in=new Scanner(System.in);
		double pi=Math.acos(-1);
		double v;
		int r,a;
		int t=in.nextInt();
		int n;
		for(int k=0;k<t;k++)
		{
			n=in.nextInt();
			v=0;
			for(int i=0;i<n;i++)
			{
				r=in.nextInt();
				a=in.nextInt();
				v=v+pi*r*r*a;
			}
            System.out.println(formateador.format(v));
		}
	}
}

File: Java-Python-C++/coj.uci.cu/2415 - Find the Wedding Cake Volume/main.cxx
/*   Problem:  Find the Wedding Cake Volume
 *   Code: 2415
 *   Site: coj.uci.cu
 *   Status: Accepted
 *   type: Geometria
 */


#include <iostream>
#include <cstdio>
#include <cmath>

using namespace std;

int main(int argc, char **argv)
{
	double pi=acos(-1);
	double v;
	int r,a,t,n;
	cin>>t;		
		for(int k=0;k<t;k++)
		{
			cin>>n;
			v=0;
			for(int i=0;i<n;i++)
			{
				cin>>r;
				cin>>a;
				v=v+pi*r*r*a;
			}
            printf("%.2f\n",v);
		}
	return 0;
}

